File: Auth_Services/Auth_Services/Models/Sala.cs
namespace Auth_Services.Models
{
    public class Sala
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public int TemPcs { get; set; } = 0; // 0 = No, 1 = Yes
        public int TemOficina { get; set; } = 0; // 0 = No, 1 = Yes
        public int IsDeleted { get; set; } = 0;
    }
}

File: AndroidApp/app/src/main/java/com/example/nts_app/network/Sala.kt
package com.example.nts_app.network

data class Sala(
    val id: Int,
    val nome: String,
    val temPcs: Int,      // 0 = No, 1 = Yes
    val temOficina: Int,  // 0 = No, 1 = Yes
    val isDeleted: Int
)